### ISSUE TITLE
Session crash during initialization

### ISSUE THREAD

**Issue by mmatera** [lcarsde/lcarswm#22]
## Description

When I call it from the terminal, lcarswm 
```
/usr/bin/lcarswm
```

the program aborts with the message:
```
INFO: ::runWindowManager::start lcarswm initialization
DEBUG: RandrHandlerFactory::init::event base: 89, error base: 147
ERROR: ::runWindowManager::unable to load settings
```

I find the same behavior both from the precompiled package https://github.com/lcarsde/lcarswm/releases/download/22.1/lcarswm_22.1_amd64.deb as from the locally compiled master branch of the project.

# Environment
OS: Ubuntu Mate - 21.10
Kernel: Linux 5.13.0-39-generic #44-Ubuntu SMP Thu Mar 24 15:35:05 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
Processor: Intel i7 - 16GB RAM

**Comment by atennert** [comment]
Hello @mmatera,

lcarswm is supposed to be started by the display manager (via the login page). There you should be able to select lcarswm and run it. It will not run when another window manager is already active. If you login from the terminal (no window manager session open) then try to use `startx /usr/bin/lcarswm` or better `/usr/bin/startlcarswm`, which sets some environment variables, including one that might be missing in the case of this error.

Does that fix the problem?

**Comment by mmatera** [comment]
I try both: to start directly from Lightdm (the default session manager in ubuntu mate) and by using `startx` from a terminal. With Lightdm it seems that also fails, because it started the default gnome wm. On the other hand, using startx from a terminal (after disabling the session manager) and (with startlcarswm) it shows the background and the mouse but didn't respond to the keyboard commands. Attached goes the log from xinit.

[log.txt](https://github.com/lcarsde/lcarswm/files/8558689/log.txt)

**Comment by atennert** [comment]
I will check this during the weekend.

**Comment by atennert** [comment]
Here is my analysis so far: I set up Ubuntu Mate in a VirtuaBox and installed LCARSDE from the deb-files. I can reproduce the issue of lightdm igoring the selection of lcarswm. I have no clue at the moment why it is doing that. I have lightdm and lcarsde on Ubuntu 20.04 and it's working fine. It seems lightdm has some issue maybe with the way that lcarswm is registered. I'll continue to look into that.

If I'm at the login window and switch to the terminal with Ctrl+Alt+F1, log in there and then run `startx startlcarswm`, then it's running fine, except for the annoying terminal window in the top left. Mouse and Keyboard are working as well, but it all needs to be configured.

**Comment by atennert** [comment]
While it is still unclear as to why it doesn't work with lightdm, I can now confirm that it works with gdm3. It might still not work with version 21.10 because the packages were built for 20.04 but soon the packages will be available for 22.04.

**Comment by atennert** [comment]
Working now for me manually compiled for 22.04 and with the updated desktop file
